pkg/server/nfs/nfs_server.go

```diff
@@ -65,6 +65,15 @@ Export_defaults
 #    Pseudo = /;
 #    FSAL { Name = VFS; }
 #}
+
+LOG {
+	Default_Log_Level = INFO;
+	Facility {
+		name = FILE;
+		destination = "/dev/stdout";
+		enable = active;
+	}
+}
```

Comment:
1. You can combine the LOG block and the above commented one.
2. `/dev/stdout` is the symbolic link of `/proc/self/fd/1`. It means the `ganesha.nfsd`'s log messages will be redirected to the stdout of `ganesha.nfsd` which is not the root process. 
kubelet only captures the `pid 1`'s stdout and stderr, so we should redirect to `/proc/1/fd/1` here.

Reply:
@derekbit thank you. I will combine the LOG lock and modify the log destination path as `/proc/1/fd/1`